### DISCUSSION TITLE
macOS 15.0 Sequoia and adl IMKClient_Legacy IMKInputSession_Legacy

### DISCUSSION THREAD

**Discussion by pol2095** [airsdk/Adobe-Runtime-Support#3475] [Q&A]
Hello,

when I launch `adl` on macOS 15.0 Sequoia I have this message :
```
2024-09-18 15:02:09.359 adl[1775:57676] +[IMKClient subclass]: chose IMKClient_Legacy
2024-09-18 15:02:09.359 adl[1775:57676] +[IMKInputSession subclass]: chose IMKInputSession_Legacy
```

Thanks

**Comment by ajwfrost** [comment]
Yes, seems to be a pretty common thing across a lot of applications! We can look into it, I'm not sure it's impacting any behaviour at the moment..?

thanks

**Comment by piotrzarzycki21** [comment]
I used to run in debug and non debug Air application from Moonshine IDE. Now after updating to Sequoia it is not possible anymore. It's just closing with above errors.

**Reply by ajwfrost** [reply]
We're using Moonshine okay with Sequoia, am wondering if there's anything particular going on in your app to cause this problem.. maybe permissions-related or similar? Do you have microphone/camera access etc? Are you able to reproduce it with a new blank ActionScript project created by Moonshine? And what do you see in the console output?

thanks

**Comment by pol2095** [comment]
On macOS 15.1, I have another message in addition :
```
CoreText registered PingFang for compatibility.
```
